src/components/OpenWith/OpenWithButton.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @flow
+ * @file Preview Open With button
+ * @author Box
+ */
+
+import * as React from 'react';
+import Button from 'box-react-ui/lib/components/button/Button';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import WithTooltip from './WithTooltip';
+import ICON_FILE_MAP from './IconFileMap';
+import messages from '../messages';
+
+type Props = {
+    extension: ?string,
+    displayIntegration: ?Integration,
+    onClick: ?Function,
+    tooltipText?: string | Object,
+    isDisabled: boolean,
+    isLoading: boolean,
+    icon?: string,
+};
+
+const INTEGRATION_ICON_CLASS = 'integration-icon';
```

Comment:
What's the value of having this as a constant? It isn't exported and seems to only be used once.

Reply:
no value, I thought I was going to use it in more than one place.